tests/integrationv2/test_external_psk.py

```diff
@@ -0,0 +1,183 @@
+import copy
+import os
+import pytest
+import time
+
+from configuration import available_ports
+from common import ProviderOptions, Protocols, data_bytes, Ciphers
+from fixtures import managed_process
+from providers import S2N, OpenSSL
+from utils import invalid_test_parameters, get_parameter_name
+
+shared_psk_identity = '2c035d829359ee5ff7af4ec900000000262a6494dc486d2c8a34cb33fa90bf1b00'\
+                      '70ad3c498883c9367c09a2be785abc55cd226097a3a982117283f82a03a143efd3'\
+                      'ff5dd36d64e861be7fd61d2827db279cce145077d454a3664d4e6da4d29ee03725'\
+                      'a6a4dafcd0fc67d2aea70529513e3da2677fa5906c5b3f7d8f92f228bda40dda72'\
+                      '1470f9fbf297b5aea617646fac5c03272e970727c621a79141ef5f7de6505e5bfb'\
+                      'c388e93343694093934ae4d357fad6aacb'
```

Comment:
This is sort of a nit, but this identity has an extra 4 bytes appended to the end. "fad6aacb" is the ticket_age_add value from the known handshake traces. So technically, the psk identity is everything before those four bytes.

Reply:
In this case it does not matter, as the psk_identity can be any value. I could update to match the other parts of s2n codebase.

Reply:
It does not matter. But I think that if we're going to use known values from the handshake traces as a psk identity, we should use the correct trace.